types/InvocationContext.d.ts

```diff
@@ -90,6 +90,12 @@ export declare class InvocationContext {
      * For example, this will be undefined for http and timer triggers because you can find that information on the request & timer object instead
      */
     triggerMetadata?: TriggerMetadata;
+
+    /**
+     * The options used when registering the function
+     * NOTE: This value may differ slightly from the original because it has been validated and defaults may have been explicitly added
+     */
+    options: EffectiveFunctionOptions;
```

Comment:
`EffectiveFunctionOptions` has `extraInputs` and `extraOuputs` as optional, when in reality, they're always there, even if initialized to empty arrays (as I think they should). Can we have another type for this that expresses that?

Reply:
Done